tutorials/RichDisplay.md

```diff
@@ -0,0 +1,91 @@
+{@link RichDisplay} allows you to create a paginated menu that users of your bot will be able to browse by using reaction-based navigation.
+
+The most simple working example can achieved by this code:
+
+```javascript
+const { Command, RichDisplay } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, { description: 'Test RichDisplay' });
+	}
+
+	async run(msg) {
+		return new RichDisplay()
+			.addPage(new Embed())
+			.run(await msg.send('Loading...'));
```

Comment:
I am pretty sure this wouldn't even load any reactions, as there is only one page.

Reply:
It should although show the page, just not the emotes.
I will add an extra page just to make a working example with navigation.